addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5917,15 +5897,24 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
 
 /**
  * @param {Element} containerEl
+ * @param {Boolean} labelIsDimension - Optional display imgsize attribute instead of animated
  * @returns {Element}
  */
-const _addAnimatedShapeLabel = function addAnimatedShapeLabel(containerEl) {
+const _addAnimatedShapeLabel = function addAnimatedShapeLabel(containerEl, labelIsDimension = false) {
     const labelEl = document.createElement('span');
     labelEl.classList.add('o_we_shape_animated_label');
-    const labelStr = _t("Animated");
-    labelEl.textContent = labelStr[0];
+    let labelStr = _t("Animated");
     const spanEl = document.createElement('span');
-    spanEl.textContent = labelStr.substr(1);
+    if (labelIsDimension) {
+        const dimensionIcon = document.createElement('i');
+        labelStr = _t(containerEl.dataset.imgSize);
```

Comment:
Translating a variable here :grimacing: Probably means it is not translated unless you defined those dataset values as translatable, does not seem to be the case. Do we need to translate those?

Reply:
My bad indeed, the data-img-size is a ratio, for this case it doesn't need a translation since it's simply a ratio (eg 1:1) I'm removing the translation